src/vscode-bicep/package.json

```diff
@@ -136,44 +136,44 @@
       "editor/title": [
         {
           "command": "bicep.showVisualizerToSide",
-          "when": "editorLangId == bicep",
+          "when": "editorLangId == bicep && commandsRegistered",
```

Comment:
>commandsRegistered [](http://example.com/codeflow?start=44&length=18)

Is this variable scoped to just our exension or does it become a global?

Reply:
It's just scoped to our extension.